cmd/containerd-shim-runhcs-v1/options/next.pb.txt

```diff
@@ -1,221 +0,0 @@
-file {
```

Comment:
@kevpar do we still need these files for something?

Reply:
I don't think we need these files. It looks like they are used as part of documenting old API versions in containerd.

Reply:
@kevpar Awesome

Reply:
If you're removing it, though, you should also remove the descriptor entry from Protobuild.toml.